cookietemple/create/template_creator.py

```diff
@@ -221,7 +221,7 @@ def prompt_general_template_configuration(self, dot_cookietemple: OrderedDict):
                                                                  to_get_property='version')
 
         # make sure that the version has the right format
-        while not re.match(r'(?<!\.)\d+(?:\.\d+){2}(?:-SNAPSHOT)?(?!\.)', poss_vers) and not dot_cookietemple:
+        while not re.match(r'(?<!.)\d+(?:\.\d+){2}(?:-SNAPSHOT)?(?!.)', poss_vers) and not dot_cookietemple:
```

Comment:
Do you have to adapt the regex in the _check_version_match function of the linter?